lib/cascading-config-array-factory.js

```diff
@@ -484,7 +486,7 @@ class CascadingConfigArrayFactory {
 
             // Validate rule settings and environments.
             const validator = new ConfigValidator({
-                builtInRules: configArrayFactory.builtInRules
```

Comment:
`configArrayFactory` stores `builtInRules` in `internalSlotsMap`, so this was undefined.